pkg/ruler/ruler.go

```diff
@@ -273,6 +281,13 @@ type MultiTenantManager interface {
 //	|                              +-----------------+              |
 //	|                                                               |
 //	+---------------------------------------------------------------+
+
+type ruler interface {
```

Comment:
Is this nencessary

Reply:
Please see https://github.com/cortexproject/cortex/pull/6129#discussion_r1704241686